### PR TITLE
FTS page fixes

### PR DESCRIPTION
**Pull request by sirainen**
Not perfect, but makes more sense than before..

### CODE REVIEW

File: source/configuration_manual/fts/squat.rst [hunk 1]
```diff
@@ -0,0 +1,150 @@
+.. _fts_backend_squat:
+
+Squat FTS Engine
+================
+
+NOTE: The Squat code is quite slow for large mailboxes. There are also a
+few bugs that are unlikely to be fixed. It's recommended to use
+:ref:`fts_backend_solr` instead.
```
Comment (cmouse) [inline source/configuration_manual/fts/squat.rst:8]: please indicate here that SQUAT is obsolete, should not be used and is going to be removed.

File: source/settings/plugin/fts-plugin.rst [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
++---------------+---------------------------------------+
+| pt            | Portuguese                            |
++---------------+---------------------------------------+
+| ro            | Romanian                              |
++---------------+---------------------------------------+
+| ru            | Russian                               |
++---------------+---------------------------------------+
+| sv            | Swedish                               |
++---------------+---------------------------------------+
+
```
Comment (thuovila) [inline source/settings/plugin/fts-plugin.rst:321]: Turkish (tr) is missing I think.

File: source/settings/plugin/fts-plugin.rst [hunk 3]
```diff
@@ -0,0 +1,453 @@
...
+  | no            | Norwegian (Bokmål & Nynorsk detected) |
+  +---------------+---------------------------------------+
+  | pt            | Portuguese                            |
+  +---------------+---------------------------------------+
+  | ro            | Romanian                              |
+  +---------------+---------------------------------------+
+  | ru            | Russian                               |
+  +---------------+---------------------------------------+
+  | sv            | Swedish                               |
+  +---------------+---------------------------------------+
```
Comment (thuovila) [inline source/settings/plugin/fts-plugin.rst:201]: Should add Turkish and French here or somehow combine this with supported languages table.

File: source/settings/plugin/fts-plugin.rst [hunk 4]
```diff
@@ -0,0 +1,453 @@
...
+  +---------------+----------+
+  | Language Code | Language |
+  +===============+==========+
+  | en            | English  |
+  +---------------+----------+
+  | fi            | Finnish  |
+  +---------------+----------+
+  | fr            | French   |
+  +---------------+----------+
+
```
Comment (thuovila) [inline source/settings/plugin/fts-plugin.rst:159]: I think there are many more stop word files now.

File: source/settings/plugin/fts-plugin.rst (hunk 2 above)
Comment (sirainen) [reply source/settings/plugin/fts-plugin.rst:291]: synced/merged languages

**Pull request merged**